src/hotspot/share/opto/type.cpp

```diff
@@ -992,6 +992,9 @@ class VerifyMeet {
 void Type::check_symmetrical(const Type* t, const Type* mt, const VerifyMeet& verify) const {
   Compile* C = Compile::current();
   const Type* mt2 = verify.meet(t, this);
+
+  // Verify that:
+  //      this meet t == t meet this
```

Comment:
Found these comments useful while verifying the type system changes.

Reply:
Nice!